pkg/ccl/backupccl/backup_test.go

```diff
@@ -2364,9 +2364,9 @@ CREATE TABLE d.t1 (x d.farewell);
 RESTORE DATABASE d FROM 'nodelocal://0/rev-history-backup'
   AS OF SYSTEM TIME %s
 `, ts1))
-			sqlDB.ExpectErr(t, `pq: type "d.public.farewell" already exists`,
+			sqlDB.ExpectErr(t, `pq: type "farewell" already exists`,
```

Comment:
This message change and others like it are caused by having unified the descriptor collision logic in `CheckObjectCollision` in `catalogkv`.